cmd/admin-prometheus-metrics.go

```diff
@@ -42,7 +42,7 @@ USAGE:
   {{.HelpName}} TARGET [METRIC-TYPE]
 
 METRIC-TYPE:
-  valid values are ['cluster', 'node', 'bucket']. Defaults to 'cluster' if not specified.
+  valid values are ['cluster', 'node', 'bucket', 'resource']. Defaults to 'cluster' if not specified.
```

Comment:
Add an entry for cmd/admin-prometheus-generate.go as well to generate scrape config for `resource` metrics. Later if we create grafana dashboard for visualization of these metrics, it would be useful.

Reply:
thanks, done